ecmd-core/capi/ecmdClientCapi.H

```diff
@@ -5183,6 +5183,22 @@ uint32_t ecmdRelatedTargets(const ecmdChipTarget & i_target, const std::string i
  TARGET STATES : Unused<br>
 */
 uint32_t ecmdQueryFileLocationHidden(ecmdChipTarget & i_target, ecmdFileType_t i_fileType, std::list<std::pair<std::string, std::string> > & o_fileLocations, std::string & io_version);
+
+/**
+ @brief Query the location of a specific file type for the selected target
+ @param i_target Struct that specifies the target to operate on (see target depth and states below)
+ @param i_fileType Enum that specifies which type of file you are looking for scandef/spydef/arraydef
+ @param o_fileLocations Return list of locations with full path and filename to location.
+ @param io_version Specify version number of file; set to version number by plugin
+ @retval ECMD_SUCCESS if successful
+ @retval ECMD_UNKNOWN_FILE if unable to find requested file
+ @retval nonzero if unsuccessful
+
+ NOTE : This function does not affect ring caching<br>
+ TARGET DEPTH  : cage, node, slot, pos (as applicable based on i_fileType)<br>
+ TARGET STATES : Unused<br>
+*/
+uint32_t ecmdQueryFileLocationHidden2(const ecmdChipTarget & i_target, ecmdFileType_t i_fileType, std::list<ecmdFileLocation> & o_fileLocations, std::string & io_version);
```

Comment:
This should get added at the end of the file so the generated enum values remain the same otherwise.